amp-pwa-reader/gulpfile.js

```diff
@@ -118,64 +118,65 @@ function clean() {
 }
 
 function modularizeJS(name) {
-  return "\nmodule.exports = " + name + ';';
+  return '\nmodule.exports = ' + name + ';';
```

Comment:
Thanks for that, @DavidScales !  I thought double quotes were required for a newline.  Now I'm not sure why I thought that.